src/logic/contracts/safeContracts.js

```diff
@@ -10,19 +10,46 @@ import { calculateGasOf, calculateGasPrice } from '~/logic/wallets/ethTransactio
 import { ZERO_ADDRESS } from '~/logic/wallets/ethAddresses'
 
 export const SENTINEL_ADDRESS = '0x0000000000000000000000000000000000000001'
+export const multiSendAddress = '0xB522a9f781924eD250A11C54105E51840B138AdD'
 
 let proxyFactoryMaster
 let safeMaster
 
 const createGnosisSafeContract = (web3: any) => {
-  const gnosisSafe = contract(GnosisSafeSol)
+  const gnosisSafeSol = { ...GnosisSafeSol }
+
+  if (!Object.keys(gnosisSafeSol.networks).length) {
+    gnosisSafeSol.networks = {
+      4: {
```

Comment:
I think we're missing the `mainnet` support here

Reply:
done